src/app/components/core/control-dispatcher/api-flow/add-edit-api-flow/add-edit-api-flow.component.html

```diff
@@ -419,7 +416,7 @@
                     <div class="field">
                         <div class="columns">
                             <div class="column is-2">
-                                <div *ngIf="editMode && allowExecute" class="field has-addons is-fullwidth">
+                                <div *ngIf="editMode && af.permitted" class="field has-addons is-fullwidth">
```

Comment:
All conditionals stuff with edit mode for permitted needs proper testing; for each role based user; and not auth mode.